redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -284,6 +284,12 @@ inline void
                             std::to_string(enabledPropVal),
                             "HardwareIsolation");
                     }
+                    else if (strcmp(dbusError->name,
+                                    "xyz.openbmc_project.Common.Error."
+                                    "InsufficientPermission") == 0)
+                    {
+                        messages::resourceCannotBeDeleted(aResp->res);
```

Comment:
Shouldn't this return a Forbidden  / 403 ?

Reply:
I think we are sending 403 from the **resourceCannotBeDeleted**, right?

redfish-core/src/error_messages.cpp:345

```
void resourceCannotBeDeleted(crow::Response& res)
{
    res.result(boost::beast::http::status::forbidden);
    addMessageToErrorJson(res.jsonValue, resourceCannotBeDeleted());
}
```